### ISSUE TITLE
Data Explorer Integration Crashes in Desktop

### ISSUE THREAD

**Issue by MSeal** [nteract/archived-desktop-app#5139]
**Application or Package Used**
State which application or package this affects. For example: nteract desktop, nteract web, nteract on jupyter, etc.

nteract desktop

**Describe the bug**

Using specific graph options for a data-explorer output causes it to crash.

**To Reproduce**

1. Open a python3 notebooks
2. Add 
```
%pip install pandas vega_datasets
```
to the first cell and run it
3. Add
```
from vega_datasets import data
import pandas

pandas.set_option('display.html.table_schema', True)
```
and run in the next cell
4. Add
```
data.cars()
```
5. Click on Summary Diagram on the available display outputs of data explorer.

Printed stack track (for what it's worth)
```
 in p
    in div
    in t
    in styled.div
    in div
    in t
    in div
    in Unknown
    in div
    in Unknown
    in t
    in div
    in Unknown
    in div
    in Unknown
    in t
    in OrdinalFrame
    in div
    in ResponsiveOrdinalFrame
    in div
    in t
    in styled.div
    in div
    in t
    in styled.div
    in Viz
    in div
    in t
    in styled.div
    in div
    in k
    in Unknown
    in Connect(Component)
    in s
    in div
    in c
    in Connect(c)
    in div
    in m
    in UndoableDelete
    in div
    in BareUndoableCellDelete
    in UnstyledUndoableCellDelete
    in t
    in UndoableCellDelete
    in p
    in Connect(p)
    in div
    in s
    in Connect(s)
    in div
    in t
    in styled.div
    in div
    in t
    in styled.div
    in div
    in m
    in DropTarget(m)
    in DragSource(DropTarget(m))
    in Connect(DragSource(DropTarget(m)))
    in i
    in div
    in s
    in Connect(s)
    in div
    in d
    in Connect(d)
    in u
    in Connect(u)
    in u
    in Connect(u)
    in b
    in DragDropContext(b)
    in Unknown
    in u
    in t
    in M
```

**Expected behavior**

The graph that initially renders doesn't crash and stays active.

**Screenshots**

![react-crash](https://user-images.githubusercontent.com/1863892/83687764-1773bc80-a5a1-11ea-915d-99a5fbbfc86f.png)

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04
- Browser Electron
- Version 0.23.3

**Additional context**

**Comment by captainsafia** [comment]
The React error associated with this is:

```
Objects are not valid as a React child (found: [missing argument]). If you meant to render a collection of children, use an array instead.
```

I'm unable to repro the error using the sample notebook that you shared. Can you repro this on the dev instance?

**Comment by MSeal** [comment]
Yes I can reproduce in dev. @emeeks fyi

**Comment by emeeks** [comment]
I'm sure it's a date object based on the details that's not being toStringed somewhere. Will follow up with Matt and knock it out.

**Comment by emeeks** [comment]
This is fixed in `8.2.7`
